src/PrestaShopBundle/Resources/views/Admin/Module/common.html.twig

```diff
@@ -32,7 +32,21 @@
       'configurationPage': '{{ path('admin_module_configure_action', {'module_name': ':number:'})|e('js') }}',
       'moduleImport': '{{ path('admin_module_import')|e('js') }}',
       'notificationsCount': '{{ path('admin_module_notification_count')|e('js') }}',
+      'maintenancePage': '{{ path('admin_maintenance')|e('js') }}',
     };
+
+    var moduleTranslations = {
+      'singleModuleModalUpdateTitle': '{{ "Are you sure you want to upgrade this module?"|trans({}, 'Admin.Modules.Notification') }}',
+      'multipleModuleModalUpdateTitle': '{{ "Are you sure you want to upgrade these modules?"|trans({}, 'Admin.Modules.Notification') }}',
+      'moduleModalUpdateCancel': '{{ "Cancel"|trans({}, 'Admin.Actions') }}',
+      'moduleModalUpdateMaintenance': '{{ "Go to maintenance page"|trans({}, 'Admin.Actions') }}',
+      'moduleModalUpdateUpgrade': '{{ "Upgrade"|trans({}, 'Admin.Actions') }}',
+      'multipleModuleModalUpdateUpgrade': '{{ "Upgrade anyway"|trans({}, 'Admin.Actions') }}',
```

Comment:
Just a question... Why "Upgrade anyway" is preceded by "**multiple**ModuleModalUpdateUpgrade" ? Because this wording is not linked with the number of modules but with the user being on maintenance mode. This wording could appear for the update of one **or** multiple modules if the user is already on maintenance mode.

I might be wrong but I just wanted to make sure to understand :)

Reply:
You're completely right, changing it